src/profile-logic/import/chrome.js

```diff
@@ -678,10 +680,28 @@ async function processTracingEvents(
       }
 
       if (profileEvent.name === 'CpuProfile') {
```

Comment:
I noticed there's a similar if condition above in the same function, where we do also set the `threadInfo.lastSeenTime` information. I think it would be good to move all of this in the if block above (especially that it's closer to the `Profile` block too). It would also be good to add a comment to remind the reader about the fact that `CpuProfile` is added by us in `wrapCpuProfileInEvent` because it took me 30min to remember that :D

Reply:
Hopefully will also get to this by early this week (or at least before this week ends).

Reply:
> I noticed there's a similar if condition above in the same function, where we do also set the `threadInfo.lastSeenTime` information. I think it would be good to move all of this in the if block above (especially that it's closer to the `Profile` block too).

Done in f1108a4d1930082749193e7f8a32eeace2c04e85.

> It would also be good to add a comment to remind the reader about the fact that `CpuProfile` is added by us in `wrapCpuProfileInEvent` ...

Done in 2982534bd25e489778c2306978c7e0b3b89aece5. The `ts` field should matter?